--- coverage.py 2014-02-03 04:33:29.000000000 +0200
+++ coverage.py 2014-02-03 04:36:33.000000000 +0200
@@ -253,6 +253,7 @@
                 src_report = {}
                 src_report['name'] = filepath
                 coverage = []
+                print filepath
                 with io.open(filepath, encoding=args.encoding) as fobj:
                     for line in fobj:
                         coverage.append(None)
